src/record.c

```diff
@@ -12,6 +12,7 @@ RecordType StopRecordType;
 Record StopRecord;
 Record WaitRecord;
 Record DummyRecord;
+Record NullRecord;
```

Comment:
I do not see where you initialize it, what promise in the if I will do GetType on NullRecord I will get nullRecordType?

Reply:
@MeirShpilraien its a static var, do we need to initialise it? if so how?